packages/wallets/src/components/WalletsCarouselContent/WalletsCarouselContent.tsx

```diff
@@ -30,6 +36,80 @@ const WalletsCarouselContent: React.FC<TProps> = ({ onWalletSettled }) => {
     // for the embla "on select" callback
     // to avoid unbinding / cleaning etc, just let it use up-to-date list
     const walletsAccountsListRef = useRef(walletAccountsList);
+    const transitionNodes = useRef<HTMLElement[]>([]);
+    const transitionFactor = useRef(0);
+
+    const numberWithinRange = ({ max, min, number }: NumberWithinRangeProps): number =>
```

Comment:
worth putting outside of component? v.minor but no point in recrating that function every time unnecesarily,
also, typically as a rule of thumb its better to use primitives instead of introducing new type, e.g.
numberWithinRange(max: number, min: number, x: number):number

Reply:
Updated